mllib/src/main/scala/org/apache/spark/ml/feature/QuantileDiscretizer.scala

```diff
@@ -107,11 +107,11 @@ private[feature] trait QuantileDiscretizerBase extends Params
  * possible that the number of buckets used will be smaller than this value, for example, if there
  * are too few distinct values of the input to create enough distinct quantiles.
  * Since 2.3.0,
- * `QuantileDiscretizer ` can map multiple columns at once by setting the `inputCols` parameter.
- * Note that when both the `inputCol` and `inputCols` parameters are set, a log warning will be
- * printed and only `inputCol` will take effect, while `inputCols` will be ignored. To specify
- * the number of bucketsfor each column , the `numBucketsArray ` parameter can be set, or if the
- *  number of buckets should be the same across columns, `numBuckets` can be set as a convenience.
+ * `QuantileDiscretizer` can map multiple columns at once by setting the `inputCols` parameter.
+ * Note that only one of `inputCol` and `inputCols` parameters can be set. If both of the
+ * `inputCol` and `inputCols` parameters are set, an Exception will be thrown. To specify the
```

Comment:
Think we can simplify to "If both `inputCol` and `inputCols` are set, ..." (since we already said in the previous sentence that only one of the parameters can be set)

Reply:
@MLnick Thank you very much for your comments. I will change these.